gulpfile.js

```diff
@@ -50,8 +52,9 @@ gulp.task('build', buildTask);
 gulp.task('package', packageTask);
 gulp.task('watch', watchTask);
 gulp.task('lint', lintTask);
+gulp.task('lint-html', lintHtmlTask);
 gulp.task('docs', docsTask);
-gulp.task('test', ['lint', 'validHTML', 'unittest']);
+gulp.task('test', ['lint', 'lint-html', 'validHTML', 'unittest']);
```

Comment:
I think htmllint do the same job as `gulp-html-validator` (unmaintained?), so we should remove the `'validHTML'` task and associated node dependencies in package.json.